docs/getstarted/index.html

```diff
@@ -1,6 +1,14 @@
 ---
 layout: docs
-title: "Get Started"
+title: "How to get started?"
 ---
-<h3>Coming soon...</h3>
-<p>By the time check out or other document pages. Thank you! :) </p>
\ No newline at end of file
+<p class="text-small">Just follow these steps and you are good to go with hexMachina framework!</p>
+
+    <div class="container">
+        <p class="text-small">1. If you haven't yet, install <a href="https://haxe.org/download/" target="_blank">Haxe Toolkit.</a></p>
+        <p class="text-small">2. Install hexMachina framework with the following command: <i class="text-info">'haxelib install hexmachina'</i></p>
+        <p class="text-small">3. Check out our <a href="https://github.com/DoclerLabs/hexHelloWorld" target="_blank">hexHelloWorld</a> repository.</p>
+        <p class="text-small">4. Have fun! :)</p>
```

Comment:
Can you describe please what to do after cloning the git repository. Maybe you can use it' readme